module-services/service-audio/AudioServiceAPI.cpp

```diff
@@ -90,16 +90,18 @@ namespace AudioServiceAPI
         return serv->bus.sendUnicast(msg, service::name::audio);
     }
 
-    bool SendEvent(sys::Service *serv, std::shared_ptr<audio::Event> evt)
+    void SendEvent(sys::Service *serv, std::shared_ptr<audio::Event> evt)
     {
         auto msg = std::make_shared<AudioEventRequest>(std::move(evt));
-        return serv->bus.sendUnicast(msg, service::name::audio);
+        serv->bus.sendMulticast(msg, sys::BusChannel::ServiceAudioNotifications);
+        // return true;
     }
 
-    bool SendEvent(sys::Service *serv, audio::EventType eType, audio::Event::DeviceState state)
+    void SendEvent(sys::Service *serv, audio::EventType eType, audio::Event::DeviceState state)
     {
         auto msg = std::make_shared<AudioEventRequest>(eType, state);
-        return serv->bus.sendUnicast(msg, service::name::audio);
+        serv->bus.sendMulticast(msg, sys::BusChannel::ServiceAudioNotifications);
+        // return true;
```

Comment:
why did you change it to void?

Reply:
nevermind - I didn't noticed the multicast

Reply:
Two reasons:
1. It is not checked anywhere what was the outcome of this function
2. Multicast doesn't return anything, so I cannot check if the message was actually sent or not